db/seeds.rb

```diff
@@ -46,6 +46,15 @@
                 institution: ugent
   end
 
+  puts 'Creating API tokens'
+
+  [zeus, staff, jelix, mart, student].each do |user|
+    token = user.username
```

Comment:
```suggestion
    token = user.token || user.username
```
This way, the first three accounts will have API tokens zeus, staff and student respectively (same as their sign-in tokens).

Reply:
Looks good to me :)